composer.json

```
@@ -145,6 +145,9 @@
                 "missing parent class": "https://git.drupalcode.org/issue/config_distro-3199197/-/commit/c312096d486caa2f01703ca7de1abf2285b6fac8.diff",
                 "Storage comparer issue": "https://gist.githubusercontent.com/tadean/4ad67c67a56d9fcec83db8f403ac58d7/raw/9a6762de5b014dffecf06c9a5b311f75c534fce4/config_distro_comparer.patch"
             },
+            "drupal/date_ap_style": {
+                "All Day Setting hides time display (3272760)": "https://www.drupal.org/files/issues/2022-03-31/date_ap_style-3272760-1.patch"
```

The test runner on drupal.org was not able to apply the patch (I think due to a missing newline at the end of the file) so I uploaded a new version.  We may want to reference that one instead to prevent any similar issues.

```suggestion
                "All Day Setting hides time display (3272760)": "https://www.drupal.org/files/issues/2022-03-31/3272760-7.patch"
```